src/__test__/__mock__/fake_api.js

```diff
@@ -0,0 +1,5 @@
+export const totalComment = (commentsNum) => commentsNum.length;
+
+export const totalNumMovies = (dataS) => dataS.length;
+
+// export { totalComment, totalNumMovies };
```

Comment:
- [ ] Please kindly remove this commented line of code as well to make your project cleaner. 😄